pyproject.toml

```diff
@@ -62,6 +229,20 @@ exclude = '''
 )/
 '''
 
+# Python source code security analyzer
 [tool.bandit]
-
 exclude = 'tests'
+
+[dependency-groups]
```

Comment:
I would have expected to find the dependency groups closer to the dependencies, not at the end of the file.

Reply:
Done. That's just where `uv add --dev` decided to put it.